fabric/src/main/kotlin/org/valkyrienskies/mod/fabric/common/ValkyrienSkiesModFabric.kt

```diff
@@ -49,4 +73,15 @@ class ValkyrienSkiesModFabric : ModInitializer {
                 }
             })
     }
+
+    private fun registerBlockAndItem(registryName: String, block: Block) {
```

Comment:
Shall i write a registration util, for addons and vs itself?

Reply:
Maybe? I feel like the Arch API does this better, but I didn't use it since we don't want VS2 to depend on the Arch API.

Reply:
Ye arch api can do that too but its wriiten for java, we can make cool kotlin registering 😏 that together with not depending on arch api. honestly depending on arch loom already hurts enough (altough i have yet to have problem with api but)